google/cloud/internal/timer_queue_test.cc

```diff
@@ -161,6 +178,27 @@ TEST(TimerQueueTest, ScheduleEarlierTimerMultipleRunner) {
   EXPECT_GT(*later_expire_time, *earlier_expire_time);
 }
 
+TEST(TimerQueueTest, ShutdownWithPendingTimersCancelledMultipleRunner) {
```

Comment:
Perhaps these "Single/Multiple" tests should have their common-ness extracted by making them tests parameterized on thread count.